components/message_queue/src/lib.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+//! Message queue component
+
+pub mod kafka;
+#[cfg(any(test, feature = "test"))]
+pub mod tests;
+
+use std::{collections::BTreeMap, result::Result};
+
+use async_trait::async_trait;
+use chrono::{DateTime, Utc};
+
+/// Topic's producer
+///
+/// In ceresdb every topic just has one partition.
+/// Because there won't be a huge number of topic(the upper bound is just about
+/// 1000).
```

Comment:
It is hard to understand these comments, and how about just removing them here?

Reply:
Done.